### PR TITLE
Post and Filter

### PR DESCRIPTION
**Pull request by FrancoBarlocco**

### CODE REVIEW

File: client/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -19,6 +21,7 @@ const routes: Routes = [
   { path: 'details/:id', component: DetailsComponent },
   { path: 'messages', component: MessagesComponent },
   { path: 'chat/:id', component: ChatComponent },
+  { path: 'filter', component: FilterComponent},
```
Comment (alfonsoLeandro) [inline client/src/app/app-routing.module.ts]: Shouldn't filters be a modal? Why did you add it to the routes array?

File: client/src/app/pages/filter/filter.component.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class="general">
+    <h1>Filtrar</h1>
+
+    <div class="control">
+        <h2>Edad</h2>
+        <ul>
+            <li><input type="radio" name="age"> &lt; 1</li>
```
Comment (alfonsoLeandro) [inline client/src/app/pages/filter/filter.component.html]: You shold have used mat-select for this, maybe remake this using ion selects after the migration

File: client/src/app/pages/filter/filter.component.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+body{
```
Comment (alfonsoLeandro) [inline client/src/app/pages/filter/filter.component.scss]: What is this doing here, there's no body in filter component html

**Comment by alfonsoLeandro** [comment]
Fix the comments I added and it's ready to merge for migration, but it has several things we should fix, after said migration

**Comment by alfonsoLeandro** [comment]
LGMT! solve conflicts and we're ready to migrate

**Review by jcabrera-ucu — APPROVED** [review]
